package.json

```diff
@@ -29,10 +29,8 @@
   "dependencies": {
     "@readme/data-urls": "^1.0.1",
     "@types/har-format": "^1.2.12",
-    "readable-stream": "^3.6.0"
-  },
-  "optionalDependencies": {
-    "formdata-node": "^4.3.2"
+    "readable-stream": "^3.6.0",
+    "undici": "^5.24.0"
```

Comment:
aren't you only pulling types from `undici`? could this be a dev-dep?

Reply:
No it's pulling the `File` API out of `undici` if we don't have it -- which will be all Node environments because native fetch doesn't ship that. But it ships a global `FormData`? 🤷